draft-ietf-quic-transport.md

```diff
@@ -1156,6 +1156,34 @@ SHOULD ignore any such packets.
 
 Servers MUST drop incoming packets under all other circumstances.
 
+### Considerations for 5-tuple routing architectures
+
+QUIC endpoints can be deployed behind a 5-tuple based routing architecture that
+delivers packets based on both the source and destination IP addresses and
+ports. When routing depends on addresses that the endpoint does not control,
+changes to the IP address or port of peers could result in packets being routed
+to a different server. The following actions could mitigate or resolve
+operational and security issues in this case:
+
+* Endpoints can use an out-of-band mechanism to deliver packets to the correct
+destination or transfer state from the original destination.
+
+* A server can request that a connection be migrated to an address that is
+unique using the preferred_address transport parameter. For example, the initial
```

Comment:
```suggestion
stable using the preferred_address transport parameter. For example, the initial
```

Reply:
If it was unique, it would be linkable, which seems sub-optimal.

Reply:
Uniqueness is correct though.  The point is that the address is *in itself* sufficient for routing, for which uniqueness is necessary.  Stability is necessary for other reasons.  

That all said, it is true that linkability is a natural consequence of this.  We need to highlight that.  Servers that use this style of demultiplexing will create linkability between paths.  That's a big deal that needs to be highlighted.

Reply:
So is the consensus here that we want to leave it with "unique"?

Reply:
It seems like the property we're intending to talk about is not uniqueness across clients, but an address that reaches that server specifically rather than a group of servers.  (Or reaches a smaller group of servers capable of internal routing based on CID, which amounts to the same thing.)  Perhaps the right way to express this is "an address that does not rely on 5-tuple routing"?

Reply:
That is imprecise.  A connection ID means that you don't rely on 5-tuple routing.  The relevant point is that the destination address of packets is unique - at least for the time that the connection is active.

Reply:
I think we settled on "unique". resolving...